authapi/permissions.py

```diff
@@ -162,3 +172,52 @@ def object_permission_set(self):
                 )
             )
         )
+
+
+class UserPermission(BaseComposedPermision):
+    '''Permissions for the UserViewSet.'''
+    def global_permission_set(self):
+        '''All users must be authenticated. Only admins can create other admin
+        users.'''
+        only_admins_create_admins = Or(
+            AllowAdmin,
+            And(
+                ObjAttrTrue(
+                    lambda r, _: r.data.get('admin').lower() != 'true'),
```

Comment:
Apparently not. For some reason `data` here is a QueryDict (https://docs.djangoproject.com/en/1.9/ref/request-response/#querydict-objects), so it returns `true`/`false` as a string. I think it might have to do with it not actually sending it as JSON, but instead as form data.. I'll look into it and check it out.